app/controllers/test/components/mentoring/mentor_conversations_lists/conversations_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Test::Components::Mentoring::MentorConversationsLists::ConversationsController < ApplicationController
```

Comment:
Thoughts on renaming `MentorConversationsLists` to `ConversationsList` as it's already scoped?

Reply:
I'm not entirely sure. The component is named as `<MentorConversationsLists />` still. Maybe we could rename the component to `<Mentoring.ConversationsList />`?

We won't need to think about this though if we go with your suggestion below to make this a controller action.

Reply:
Yeah - let's go with this: `<Mentoring.ConversationsList />`